provider-directory/src/test/resources/LocationTestResource/mock-care-sites-response.json

```diff
@@ -0,0 +1,3137 @@
+{
+    "@odata.context" : "https://dws.ppms.va.gov/v1.0/$metadata#Collection(PpmsDataWebService.Models.CareSite)",
+    "value" : [
+        {
+            "Name" : "Beacon Orthopaedics & Sports Medicine Ltd",
```

Comment:
This file seems needlessly large. Can we restrict this to just the records in use in the test?